docs/Training-Generalized-Reinforcement-Learning-Agents.md

```diff
@@ -20,7 +20,9 @@ Ball scale of 0.5          |  Ball scale of 4
 
 ## Introducing Generalization Using Reset Parameters
 
-To enable variations in the environments, we implemented `Reset Parameters`. We
+To enable variations in the environments, we implemented `Reset Parameters`.
+`ResetParameters` are `Academy.FloatProperties` that are used only when
```

Comment:
Space in `Reset Parameters`. So what differentiates these two is that FloatProperties could (technically) be used anytime but our example envs only use them at reset right? The phrasing is a bit confusing but I think that makes sense.